dist/tools/backport_pr/backport_pr.py

```diff
@@ -204,6 +204,7 @@ def main():
         # Delete worktree
         print("Pruning temporary workdir at {}".format(worktree_dir))
         _delete_worktree(repo, worktree_dir)
+    repo.branches[new_branch].set_tracking_branch(push_info[0].remote_ref)
```

Comment:
I think this should be done just after the `origin.push`. Otherwise with `--noop`:

```
UnboundLocalError: local variable 'push_info' referenced before assignment
```

Which also would be the logical equivalent of `git push -u`.